services/graph/pkg/identity/ldap.go

```diff
@@ -72,14 +72,15 @@ type LDAP struct {
 }
 
 type userAttributeMap struct {
-	displayName    string
-	id             string
-	mail           string
-	userName       string
-	givenName      string
-	surname        string
-	accountEnabled string
-	userType       string
+	displayName       string
+	id                string
+	mail              string
+	userName          string
+	givenName         string
+	surname           string
+	accountEnabled    string
+	userType          string
+	preferredLanguage string
 }
```

Comment:
Should we leave the preferredLanguage completely out of the identity backend because we do not use it there?

Reply:
done